tests/unittest_scoped_nodes.py

```diff
@@ -291,6 +291,46 @@ def test_stream_api(self) -> None:
             with open(path, "rb") as file_io:
                 self.assertEqual(stream.read(), file_io.read())
 
+    def test_with_singleline_docstring(self) -> None:
+        data = """
+            '''Hello World'''
+            foo = 1
+        """
+        module = builder.parse(data, __name__)
+        if PY38_PLUS:
+            assert isinstance(module.doc_node, nodes.Const)
+            assert module.doc_node.lineno == 2
+            assert module.doc_node.col_offset == 0
+            assert module.doc_node.end_lineno == 2
+            assert module.doc_node.end_col_offset == 17
+        else:
+            assert module.doc_node is None
```

Comment:
Only `doc_node.end_lineno` and `doc_node.end_col_offset` should be `PY38_PLUS` exclusive.

Reply:
This depends on how we resolve https://github.com/PyCQA/astroid/pull/1276#discussion_r767055173